packages/core/helper-plugin/src/components/tests/RelativeTime.test.tsx

```diff
@@ -1,3 +1,4 @@
+// TODO: Needs to be reworked with proper typescript implementation. Current typescript port works but should only be used as a temporary patch
```

Comment:
I was skeptical but I actually agree, this test needs a complete rewrite, it mocks too many things to actually be useful